diesel_codegen/src/embed_migrations.rs

```diff
@@ -16,7 +16,7 @@ pub fn derive_embed_migrations(input: syn::MacroInput) -> quote::Tokens {
 
     let options = get_options_from_input(&input.attrs, bug);
     let migrations_path_opt = options.map(|o| get_option(&o, "migrations_path", bug));
```

Comment:
Is there a way to pass the lifetime of `options` to `get_option` and use that as the lifetime on the returned `str`? I couldn't get that to work, which is why I switch the return value of `get_option` to `String` (and all the associated changes `str` to `String`).